src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -216,7 +216,7 @@ public void Add_PrefixAlreadyRegisteredAndStarted_ThrowsHttpListenerException()
 
         [Theory]
         [InlineData("http://microsoft.com/")]
-        [InlineData("http://[]/")]
+        //Managed implementation allows this. Is this a bad test? [InlineData("http://[]/")]
```

Comment:
The docs give some clues so I guess I could write a parser for this - not sure how well Uri.CheckHostName or some equivalent would work with wild cards and the like.